Add Juanita's introductory blogpost (#179)
pyOpenSci/pyopensci.github.io: _posts/2023-05-20-introduction-juanita.md

    <a href="/images/introduction-juanita/leah-pyopensci.jpeg">
    <img src="/images/introduction-juanita/leah-pyopensci.jpeg" style="max-width:90%" alt="Two women wearing sunglasses, enjoying a sunny day.">
    </a>
    <figcaption> I was thrilled to recently meet Leah, the Executive Director of pyOpenSci, in person. I'm excited to work with her on this new journey and eager to learn from her experience with open source communities.
    </figcaption>
</figure>

Recently, I joined pyOpenSci, an organization that resonates deeply with my values. Their commitment to enhancing the scientific Python ecosystem through a rigorous [peer review process](https://www.pyopensci.org/software-peer-review/about/intro.html) aligns perfectly with my own aspirations. This process not only recognizes the hard work of scientists contributing to open Python tools, but also promotes standardization and increased visibility for these packages. I am excited to bring my past experiences to the table, helping to strengthen community building and outreach within this incredible project while eagerly learning from the talented individuals who are part of it.

## Outro: Gratitude

> As a community manager at both Scientific Python and pyOpenSci, I have witnessed the wonders of the Scientific Python ecosystem. I have encountered countless remarkable individuals and witnessed the incredible projects they have created using the available tools. In my opinion, unifying the community and promoting project integration is the key to unlocking the full potential of this ecosystem. It's not just about coding; it's about coming together, sharing knowledge, and supporting one another. I am grateful for the opportunity to contribute to this vision and help build an extraordinary community around the scientific Python ecosystem while walking this journey alongside amazing people. 🫶

Reviewer: @juanis2112 this is really good. i wanted to clarify the position because again we have a full time community manager coming on board so i don't want any confusion as she has been hired. So i just adjusting the language here and also made scientific Python lowercase to avoid confusion (which already exists!) because pyOpenSci, the scientific Python broader community and the Scientific Python project. If you are ok with this we can merge this and then merge the entire blog. just let me know.

<figure>
    <a href="/images/introduction-juanita/scientific-python.png">
    <img src="/images/introduction-juanita/scientific-python.png" style="max-width:100%" alt="A group of people standing in front of the AT&T Hotel and Conference Center at the University of Texas campus in Austin, Texas wearing blue and black shirts with the Scientific Python logo. ">
    </a>

>     <figcaption>This is a picture of the people who attended our <a href="https://www.youtube.com/watch?v=R9hNnhpMgTE&t=2s"> Scientific Python community outreach BOF session </a> at SciPy 2022. We gave everyone who attended Scientific Python shirts! It was an amazing discussion with people from a lot of different projects within the Scientific Python ecosystem.

Reviewer: ```suggestion
    <figcaption>This is a picture of the people who attended the <a href="https://www.youtube.com/watch?v=R9hNnhpMgTE&t=2s"> Scientific Python community outreach BOF session </a> at SciPy 2022. We gave everyone who attended Scientific Python shirts! It was an amazing discussion with people from a lot of different projects within the Scientific Python ecosystem.
```

---
layout: single
title: "🎶 My Journey Through Scientific Python: The Lyrics 🎶
 "

> excerpt: "Hi there, I'm Juanita. I recently joined pyOpenSci as a community lead to help with community building and outreach and this blogpost tells the story of how I got here. "

Reviewer: This is what search engines use to post information about this post. so i suggest a bit more straight forward excerpt that helps a reader know what this content is about!
```suggestion
excerpt: "Juanita Gomez recently joined pyOpenSci as a community lead to help with community building and outreach. Learn more about Juanita's path to becoming a leader in community management in the scientific Python ecosystem. "
```

> author: "Juanita Gomez"
> author: "Juanita Gomez"
> permalink: /blog/journey-through-scientific-python.html

Reviewer: ```suggestion
author: "Juanita Gomez"
permalink: /blog/journey-through-scientific-python.html
```

header:
  overlay_color: "#DD751C"
categories:
  - blog-post
  - highlight
  - community
toc: false
comments: true
---

## Introduction: About Me

> Hello, everyone! I'm Juanita, and I'm excited to share my story with you. I'm currently a computer science PhD student at UC Santa Cruz and the community manager of the Scientific Python project (which you will hear more about in a bit 👀). I’m originally from Colombia, and I completed my undergraduate studies in pure mathematics at Universidad Javeriana while also taking some music courses. I'm a passionate singer, and my favorite color is orange 🧡. I recently joined pyOpenSci as a community lead to help with community building and outreach (super excited about it!), so today I want to share my journey through my involvement in open-source scientific Python projects, which is basically how I got here. 😊

Reviewer: let's add a link to the scientific python home page here!

<figure>
    <a href="/images/introduction-juanita/juanita-guitar.png">

>     <img src="/images/introduction-juanita/juanita-guitar.png" style="max-width:100%" alt="A woman sitting on top of a hill in the dessert holding a guitar.">

Reviewer: ```suggestion
    <img src="/images/introduction-juanita/juanita-guitar.png" style="max-width:100%" alt="A woman sitting on top of a hill in the desert holding a guitar.">
```

    </a>
    <figcaption>This is a picture of me in the middle of a dessert <a href="https://www.youtube.com/watch?v=lFxc5NZDjuA">  playing my favorite song </a>. One of the reasons I love this picture (besides the amazing landscape) it's because it shows how happy it makes me singing. 
    </figcaption>
</figure>

## Verse 1: The Spark

> One of my earliest contributions to the open-source world was with the [Spyder IDE](https://www.spyder-ide.org) when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.

Reviewer 2: If you need it... [SVG](https://github.com/spyder-ide/spyder/blob/master/img_src/spyder.svg). [PNG](https://github.com/spyder-ide/spyder/blob/master/img_src/spyder.png)

> One of my earliest contributions to the open-source world was with the [Spyder IDE](https://www.spyder-ide.org) when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.

Reviewer 2: Our standard short-form descriptor for Spyder is "the scientific Python development environment", since it pretty succinctly (and hopefully clearly) describes what it is/what it does, and is the origin of the name. So you could say:

```suggestion
One of my earliest contributions to the open-source world was with [Spyder](https://www.spyder-ide.org), the scientific Python development environment, when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.
```

If you want a screenshot, you could use [this one](https://github.com/spyder-ide/spyder/blob/master/img_src/screenshot.png), the (relatively) up to date hero on our web page and docs that Juanita herself took (her username is actually visible in the file paths).
